ultralytics/cfg/__init__.py

```diff
@@ -546,6 +546,75 @@ def handle_yolo_settings(args: List[str]) -> None:
         LOGGER.warning(f"WARNING ⚠️ settings error: '{e}'. Please see {url} for help.")
 
 
+def handle_yolo_info(args: List[str]) -> None:
+    """Handles YOLO info command-line interface (CLI) commands."""
+    from ultralytics import NAS, RTDETR, SAM, YOLO, FastSAM
+
+    nl = "\n"
+    delim = "-" * 80
+    choose = {
+        "yolo": YOLO,
+        "rtdetr": RTDETR,
+        "fastsam": FastSAM,
+        "mobile_sam": SAM,
+        "sam_": SAM,
+        "sam2_": SAM,
+        "yolo_nas": NAS,
+    }
+    skip = {"model", "ema", "optimizer", "train_results"}
+
+    def _obj_or_dict(x: Union[object, Dict]) -> str:
+        """Returns a string representation of an object or creates a string from key-value pairs from a dictionary."""
+        return f"{x}" if not isinstance(x, dict) else "".join([f"\n\t{str(k)}: {v}" for k, v in x.items()])
+
+    def _classes(d: dict):
+        """Yields a formatted string representation of classes from a dictionary."""
+        for k, v in d.items():
+            yield f"\t{k}: {_obj_or_dict(v)}"
+
+    def _args_load(a: List[str]) -> Tuple[dict, str]:
```

Comment:
Let's show them in alphabetical order so it's easier to find the parameters.

Reply:
This is for the `yolo info` arguments, not the model arguments. I wasn't ordering those and think they should follow the same order they have in the model. That way the argument 'order' is only defined in a single location instead of in multiple places. I agree they should be ordered, but I think that should happen at the source of where the arguments for the model are defined instead of where they're displayed.